test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ContactDetailsFullNameControllerSpec.scala

```diff
@@ -107,71 +103,29 @@ class ContactDetailsFullNameControllerSpec extends ControllerSpec {
       }
 
       "return 400 (BAD_REQUEST) for " + formAction._1 when {
-        "user does not enter" when {
-          "first name" in {
-            authorizedUser()
-            val result =
-              controller.submit()(postRequestEncoded(FullName("", "LastName"), formAction))
-
-            status(result) mustBe BAD_REQUEST
-          }
-
-          "last name" in {
-            authorizedUser()
-            val result =
-              controller.submit()(postRequestEncoded(FullName("FirstName", ""), formAction))
-
-            status(result) mustBe BAD_REQUEST
-          }
+        "user does not enter name" in {
 
-          "both first name and last name" in {
-            authorizedUser()
-            val result = controller.submit()(postRequestEncoded(FullName("", ""), formAction))
+          authorizedUser()
+          val result = controller.submit()(postRequestEncoded(FullName(""), formAction))
 
-            status(result) mustBe BAD_REQUEST
-          }
+          status(result) mustBe BAD_REQUEST
         }
 
-        "user enters a long" when {
-          "first name" in {
-            authorizedUser()
-            val result = controller.submit()(
-              postRequestEncoded(FullName("averyveryveryveryverylongname", "LastName"), formAction)
-            )
-
-            status(result) mustBe BAD_REQUEST
-          }
-
-          "last name" in {
-            authorizedUser()
-            val result = controller.submit()(
-              postRequestEncoded(FullName("FirstName", "averyveryveryveryverylongname"), formAction)
-            )
+        "user enters a long name" in {
+          authorizedUser()
+          val result = controller.submit()(postRequestEncoded(FullName("abced" * 40), formAction))
```

Comment:
You feel comfortable asking the reader to perform `5 * 40` in their poor heads?

Reply:
Would you prefer I defined `val twoHundreadCharacterName = "abcde" * 40` and used that?

Reply:
Nah, it was a joke. 😁